packages/blockchain-wallet-v4-frontend/src/data/components/taxCenter/selectors.ts

```diff
@@ -0,0 +1,61 @@
+import { Types } from '@core'
+import { model, selectors } from 'data'
+
+import { getData as getDataBch } from '../../../scenes/Settings/Addresses/Bch/Wallets/selectors'
+import { getData as getDataXlm } from '../../../scenes/Settings/Addresses/Xlm/selectors'
+
+const BECH32_DERIVATION = 'bech32'
+const LEGACY_DERIVATION = 'legacy'
```

Comment:
Can we use DERIVATION_LIST?

`import { DERIVATION_LIST } from '@core/types/HDAccount'`

Reply:
ty. I removed this constants